src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppEcsFargate/Generated/Configurations/AutoScalingConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AspNetAppEcsFargate.Configurations
```

Comment:
We had always said we wanted to give user's the code directly so they could see how things are created and if they want easily disconnect themselves from our template and customize the deployment however they want. Switching to NuGet packages would be a major shift. It would make story potentially easier but lose a lot in the teaching/learning side.